autowarefoundation/autoware
# Feature #931: Create Unit Testing Example

## Status
**PRODUCTION / DEVELOPMENT**

## Description
The purpose of this PR is to introduce an example on how to create unit tests in Autoware.

## Steps to Test or Reproduce
- Start two sourced terminals (`source /opt/ros/kinetic/setup.bash`)
- start `roscore`
- In `Autoware/ros`:
     - `catkin_make run_tests`

Project should build and tests should run and pass. 

To run the newly added test:
- `catkin_make run_tests_points_preprocessor`
